### PR TITLE
Improve app loading background

### PR DESCRIPTION
**Pull request by LasseRosenow**
When the webapp is loading, in the current version of directus, it starts with a white background no matter if you're system is in dark mode or not. Then the loading screen after that (with the indicator) has a weird background color in dark mode instead of using the default background that the rest of the app is using.

This PR fixes both problems:
- I changed the default theme to "auto" so that at first the system theme will be used and then it will change to what ever is selected by the user. I think this is the best approach in handling dark and light mode in an equal way.
- I added a color-scheme meta tag, so that browsers know that the page supports dark mode. This way even the first paint will not be white if the system is in dark mode, so there will be no black white flickering anymore :)
- I removed the custom background from the indicator loading screen to make it more consistent with the rest of the app
- I removed the white background from the login screen, to make it compatible with auto mode (I think the login screen should not have light mode by default anyways. It makes most sense that it follows the color-scheme of the browser. If your browser is in dark mode, you would expect it to be dark. The app already does this anyways.

## Before

https://user-images.githubusercontent.com/10547444/164081608-6c40c742-3111-4f37-96e7-b4b5fb55fbec.mp4

## After

https://user-images.githubusercontent.com/10547444/164081631-fca2aee9-671f-4610-8bcc-9fc1d5261031.mp4

### CODE REVIEW

**Comment by benhaynes** [comment]
Thank you, and welcome to our contributor community! 🥳

File: app/src/app.vue [hunk 1]
```diff
@@ -115,7 +115,7 @@ export default defineComponent({
 	justify-content: center;
 	width: 100%;
 	height: 100%;
-	background: rgb(255 255 255 / 0.5);
+	/*background: rgb(255 255 255 / 0.5);*/
```
Comment (rijkvanzanten) [inline app/src/app.vue:118]:
```suggestion
```

**Review by rijkvanzanten — APPROVED** [review]

File: app/src/app.vue (hunk 1 above)
Comment (LasseRosenow) [reply app/src/app.vue]: Oh sry that I missed that

Comment (rijkvanzanten) [reply app/src/app.vue]: No prob! Easy tweak 🙂

**Comment by LasseRosenow** [comment]
What about applying a white background color to the logo on the login screen? This way it would not be broken in dark mode, it has an opacity layer

**Comment by LasseRosenow** [comment]
@basecde 

I like your idea of a preference to control if public pages should follow the browser / system theme or not.

As you say it solves both our use cases, it's a very good and also elegant solution I think. 

👍👍

**Comment by benhaynes** [comment]
I agree on all of this. We'll see what @rijkvanzanten says, but I think a simple env flag to toggle this might be best.

**Comment by rijkvanzanten** [comment]
Env vars aren't available on the client side, so it would actually be easier to implement to make this a flag under Directus settings. That would also tie into the overal theming discussion a little nicer I thing

**Comment by benhaynes** [comment]
Good point! CC @AustinPhillipTaylor

**Comment by AustinPhillipTaylor** [comment]
Howdy all 👋 

I actually think a good solution to this would be to default to a light theme on the login page. Once the user logs in, and any time they change their theme, we could save a cookie to their local storage indicating the selected theme. 

The login page could always check the local storage to determine if there's a theme set, and fall back to light if not.

In addition, we can add settings options for a dark theme foreground/background, and light theme foreground/background. 

I think it's a pretty simple solution, which takes care of every major use-case.

**Comment by AustinPhillipTaylor** [comment]
> The remaining issue here would be that the "first use" experience would be imperfect if your system uses dark mode.

I believe that's unavoidable. Light/dark theme selection is user-based, not instance-based. 

It would be possible to add a setting that would allow the system admin to set the default fallback, but I feel like that's a bit clunky. 

Setting the theme to `auto`, to default to the browser color scheme, is not a bad idea. Then we could fall back to light theme in the event `prefers-color-scheme` is not set (which is the current behavior for auto theme). 

Precedence would be Cookie value -> System value (auto) -> Light

**Comment by LasseRosenow** [comment]
> Precedence would be Cookie value -> System value (auto) -> Light

I like this approach :)

**Comment by rijkvanzanten** [comment]
> Ultimately my believe is that dark and light should always be treated equally and none shall be more equal than the other :D

While I fully agree with this statement, at the end of the day we have to make a decision which one we default to if all other checks and options fail. To me, that should be light mode (for very subjective design reasons)

**Pull request merged**